aws/resource_aws_workspaces_directory_test.go

```diff
@@ -515,7 +516,7 @@ data "aws_availability_zones" "available" {
 
 locals {
   region_workspaces_az_ids = {
```

Comment:
@YakDriver The basic idea behind this map providing exact AZs (with ids), where the WorkSpaces service is available. If you need to test workspace directory resource in Gov region simply add extra mapping of region name and respective AZ ids. Then `lookup` function will pick up it according to the region, where the test is run.